content/article/json-unicode-and-perl-oh-my-.md

```diff
@@ -0,0 +1,325 @@
+
+  {
+    "title"       : "JSON, Unicode, and Perl … Oh My!",
+    "authors"     : ["felipe-gasper"],
+    "date"        : "2020-01-27T07:11:44",
+    "tags"        : [],
+    "draft"       : true,
+    "image"       : "",
+    "thumbnail"   : "",
+    "description" : "A look at this popular serialization’s relationship with Perl",
+    "categories"  : "development"
+  }
+
+Consider the following code:
+
```

Comment:
You can leave this as literal if you prefer, or if you want Perl syntax highlighting do:

    ```perl
    $code
    ```